scripts/zones/Bibiki_Bay/mobs/Jagil.lua

```diff
@@ -12,7 +11,14 @@ entity.onMobDeath = function(mob, player, optParams)
 end
 
 entity.onMobDespawn = function(mob)
-    xi.mob.phOnDespawn(mob, ID.mob.SERRA_PH, 10, 3600) -- 1 hour
+    -- PH is the 5th Jagil entry in the zone
+    local jagilId  = mob:getZone():queryEntitiesByName('Jagil')[5]:getID()
+    local shenId   = mob:getZone():queryEntitiesByName('Shen')[1]:getID()
```

Comment:
Shen is not related to this PH or its NM

Reply:
Good catch 👍